firmware/boron-ins-fsm/src/imDoorSensor.h

```diff
@@ -39,13 +40,22 @@ typedef struct IMDoorID {
     unsigned char byte3;
 } IMDoorID;
 
+typedef enum
+{
+    INTERNAL,
+    EXTERNAL
+} antennaType;
+
 //************************global variable declarations***************************
 // extern os_queue_t bleQueue;
 extern os_queue_t bleHeartbeatQueue;
 
 // needs to be global because it is used in setup(), loop(), and console function
 extern IMDoorID globalDoorID;
 
+// antenna that is being used for bluetooth LE (internal or external)
+extern uint8_t bleAntennaType;
```

Comment:
I would have expected the type here to be `antennaType` instead of `uint8_t` since that is what you're going to be storing in it. 

If you want to make sure that the enum is of type `uint8_t` (which I think is a good idea), it looks like you can do something like this example:
```
// enum that takes 16 bits
enum smallenum: [std::int16_t](http://en.cppreference.com/w/cpp/types/integer)
{
    a,
    b,
    c
};
```
from here (https://en.cppreference.com/w/cpp/language/enum#Example)